bots/bot.go

```diff
@@ -0,0 +1,39 @@
+package bots
+
+import (
+	"bytes"
+	"io"
+	"net"
+)
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(addr string, msg []byte) ([]byte, error) {
+	scheme := "unix"
+	unixaddr := &net.UnixAddr{Name: addr, Net: scheme}
+
+	conn, err := net.DialUnix(scheme, nil, unixaddr)
```

Comment:
Similarly, on startup, it might be helpful to notify early if the coniks server is down.